website/docs/d/oracle_db_system_versions.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_db_system_versions"
+description: |-
+  Gets information about an existing  Oracle Dbsystem Versions.
```

Comment:
Abbreviations should not be used and the full words should be used instead. If db refers to database, please update all places of `db` to `database`